docs/registry/access-mgmt-registry/README.adoc

```diff
@@ -137,11 +139,39 @@ You can edit or remove the user roles assigned in your {registry} instances that
 .Procedure
 . In the web console, go to *{registry}* > *{registry} Instances* and click the name of the {registry} instance that you want to remove a user role for.
 . Click the *Access* tab to view the accounts and roles assigned for this instance.
-. Select the options menu (three vertical dots) next to the assigned *Role* name: 
-.. To change to a different role, click *Edit* and select the new user role, for example, *Viewer* for read-only access to this instance.  
+. Select the options menu (three vertical dots) next to the assigned *Role* name:
+.. To change to a different role, click *Edit* and select the new user role, for example, *Viewer* for read-only access to this instance.
 .. To remove the currently assigned role, click *Remove* and confirm in the dialog.
 
+[id="proc-user-account-access_{context}"]
+== Using RBAC to allow users to view other user accounts
 
+[role="_abstract"]
+As an organization administrator, you can use Role-Based Access Control (RBAC) to allow users to view other users in a group.
```

Comment:
Suggest "you can use the Role-Based Access Control (RBAC) available in the {org-name} web console to allow users..."

This is to help distinguish between the lower-level RBAC available in RHOSR, which is described earlier in this guide. We have two different RBAC layers: one in the SR data plane (for SR admins, instance owners, and org admins), and this one in the HCC for org admin users only.